mem/test.yaml

```diff
@@ -6,7 +6,7 @@ test:
       ignore: True #issue #1010 https://github.com/phoenix-rtos/phoenix-rtos-project/issues/1010
       targets:
         # mprotect: is noop on NOMMU architecture
-        exclude: [armv7m7-imxrt106x-evk, armv7m7-imxrt117x-evk, armv7m4-stm32l4x6-nucleo]
+        exclude: [armv7m7-imxrt106x-evk, armv7m7-imxrt117x-evk, armv7m4-stm32l4x6-nucleo,sparcv8leon3-generic-qemu]
```

Comment:
We want to add a target as experimental for now as we only run psh tests, please see
https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/366#discussion_r1721838618